weave

```diff
@@ -261,14 +261,14 @@ elif [ "$1" = "run" -a -z "$IS_LOCAL" ] ; then
     # we can't rely on our baked-in docker to support those arguments.
     shift 1
     if [ "$1" != "--without-dns" ] ; then
-        DNS_ARGS=$(exec_remote dns-args "$@" || true)
+        eval "$(exec_remote dns-args "$@" 2>/dev/null || true)"
```

Comment:
no we shouldn't add `test2.weave.local`. Just like we don't register the container in weavedns in that case.